recipes/refet/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "refet" %}
+{% set version = "0.3.7" %}
+{% set sha256 = "1b614ffbdaa44bcdb1869d8c2095583e6509a0cb6be952d643893a4f55dce220" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
```

Comment:
I think you might need to add `pytest-runner` to the `host` section since it is in `setup_requires` in the `setup.py`:

https://github.com/DRI-WSWUP/RefET/blob/master/setup.py#L44